src/types.tsx

```diff
@@ -9,7 +9,7 @@ export type UnflowType = {
   initialize(apiKey: string, enableLogging: boolean): null;
   sync(): null;
   setUserId(userId: string): null;
-  setAttributes({}): null;
+  setAttributes({}: { [key: string]: any }): null;
```

Comment:
does this need to be `String` and not `any`?

Reply:
Good spot